content/docs/ibm/install-kubeflow.md

```diff
@@ -116,6 +116,64 @@ components, the recommended configuration for a cluster is:
 
     Make sure all the nodes are in `Ready` state. You are now ready to install
     Istio into your cluster.
+    
+## IBM Cloud Block Storage Setup
+By default, IBM Cloud Kubernetes cluster uses [IBM Cloud File Storage](https://www.ibm.com/cloud/file-storage) as the default storageclass. File Storage is designed to run RWX (read-write multiple nodes) workloads with proper security build around it. Therefore, File Storage [does not allow `fsGroup` securityContext](https://cloud.ibm.com/docs/containers?topic=containers-security#container) which is needed for DEX and Jupyter Server.
+
+[IBM Cloud Block Storage](https://www.ibm.com/cloud/block-storage) provides a fast way to store data and
+satisfy many of the Kubeflow persistent volume requirements such as `fsGroup` out of the box and optimized RWO (read-write single node) which is used on all Kubeflow's persistent volume claim. 
+
+Therefore, we strongly recommend to set up IBM Cloud Block Storage as the default storageclass so that you can
```

Comment:
```suggestion
Therefore, we strongly recommend to set up [IBM Cloud Block Storage](https://cloud.ibm.com/docs/containers?topic=containers-block_storage#add_block) as the default storage class so that you can
```